src/worker/listeners/queuedTxListener.ts

```diff
@@ -37,6 +37,7 @@ export const queuedTxListener = async (): Promise<void> => {
         const response = await fetch(env.WEBHOOK_URL, {
           method: "POST",
           headers: {
+            "Authorization": `Bearer ${env.THIRDWEB_API_SECRET_KEY}`,
```

Comment:
adding auth make sense and i guess this add is good for now, but, I kept it without auth for now, as I didn't want to use `THIRDWEB_API_SECRET_KEY` env var becoz, if a user wants to use a different value (which is used by their webhook server where it will validate the token) then it will not work. maybe we should add new env variable for this, `WEBHOOK_AUTH_BEARER_TOKEN` or something.

Reply:
Yea probably makes sense to separate this out

Reply:
eventually all this setting/config values will be on the DB and will be able to add/update from TW Dashboard

Reply:
Thanks for reviewing! Makes sense, I updated to optional auth and using the new env var in the latest commit

Reply:
@IDubuque can you add env var this into `src/utils/env.ts` & make it optional.

Reply:
@farhanW3 added to the env schema in the latest commit

Reply:
Thanks. 

We should do some cleanup here (I wanted to do it post web-hook v0 push, so this PR would be ideal) and extract the `webhook` (fetch call) into it's own function. Otherwise, we will have to update the logic at two places.

Let me know, if can do take this up :)

Reply:
I'll take up these changes in the PR. I'd like to have auth on main before deploying engine with paper checkouts to avoid freely exposing the webhook endpoints.

Reply:
I will merge this and then do the updates I mentioned above.